content_handler/runtime_holder.cc

```diff
@@ -269,6 +269,25 @@ void RuntimeHolder::OnEvent(mozart::EventPtr event,
     pointer_data.physical_x = event->pointer_data->x;
     pointer_data.physical_y = event->pointer_data->y;
 
+    switch (pointer_data.change) {
+      case blink::PointerData::Change::kDown:
+        down_pointers_.insert(pointer_data.pointer);
+        break;
+      case blink::PointerData::Change::kCancel:
+      case blink::PointerData::Change::kUp:
+        down_pointers_.erase(pointer_data.pointer);
```

Comment:
can the C++ code miss the down event somehow? The platforms guarantee that we won't, right?